Sources/Widgets/Widgets3D/LabelWidget/behavior.js

```diff
@@ -0,0 +1,159 @@
+import macro from 'vtk.js/Sources/macros';
+
+export default function widgetBehavior(publicAPI, model) {
+  model.classHierarchy.push('vtkLabelWidgetProp');
+
+  model.textHandle = model.widgetState.getText();
+  model.moveHandle = model.widgetState.getMoveHandle();
+
+  let isDragging = null;
```

Comment:
I wonder if that should go in the model in case a subclass wants to access it (or create "protected" accessors ?)
@floryst

Reply:
If there is intention to subclass, sure. It's transient state, so it could go either way.